library-backend/library-app/src/main/java/org/internship/library/app/controller/BookController.java

```diff
@@ -32,7 +33,8 @@
      */
     @GetMapping("/{id}")
     public ResponseEntity<Book> getBook(@PathVariable String id) {
-        return ResponseEntity.status(HttpStatus.OK).body(bookService.getBook(id));
+        return ResponseEntity.status(HttpStatus.OK)
```

Comment:
What happens if the book is not found?

Consider such usescases. You should return 404.